mlflow/sklearn/__init__.py

```diff
@@ -455,6 +457,60 @@ def load_model(model_uri):
     )
 
 
+class _AutologTrainingStatus:
```

Comment:
Can we add documentation to this class and explain how it differs from things like `mlflow.utils.autologging_utils.AutologgingSessionManager`?